utils/vllm_warmup.py

```
@@ -0,0 +1,262 @@
+    candidates = []
+    if env_path:
+        candidates.append(env_path)
+    repo_root = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    candidates.append(os.path.join(repo_root, "configs", "global.yaml"))
+    for path in candidates:
+        if path and os.path.isfile(path):
+            try:
+                with open(path, "r") as f:
+                    return yaml.safe_load(f) or {}
+            except Exception:
+                pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)